Source/JavaScriptCore/bytecode/BytecodeList.rb

```diff
@@ -1951,6 +1951,22 @@
         typeIndex: unsigned,
     }
 
+op :array_get_s,
```

Comment:
Apparently landing both this PR and https://github.com/WebKit/WebKit/pull/2065 will trigger issues with wasm opcode id limits. A potential way to avoid this temporarily is to delete these extra opcodes for `get_s` and `get_u` and instead pass an argument that just flags the type of behavior for `array.get` that is desired.

Reply:
These opcodes are now removed, so this PR should not add any new LLInt opcodes.